ci/terraform/oidc/authentication_callback.tf

```diff
@@ -0,0 +1,69 @@
+module "frontend_api_authentication_callback_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "frontend-api-authentication-callback-role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = [
+    aws_iam_policy.audit_signing_key_lambda_kms_signing_policy.arn,
+    aws_iam_policy.dynamo_user_read_access_policy.arn,
```

Comment:
I'm not sure why it would need these two relating to user tables? But on the other hand, I do think it might need some extra permissions in order to store the results of the Userinfo response from Authentication...?

Reply:
There's a policy required for the `authentication-callback-userinfo` which is being created in AUT-1660. So I'll hold off merging this PR until AUT-1660 had been deployed